loleaflet/css/notebookbar.css

```diff
@@ -227,6 +227,7 @@
 
 .hasnotebookbar .ui-content.unobutton.selected {
 	border-radius: 2px;
+	box-shadow: 0px 0px 0px 3px var(--gray-bg-color);
```

Comment:
instead of adding this

Reply:
add after the following selector `.unotoolbutton.notebookbar:not(.disabled):hover, #clearFormatting.notebookbar div img:hover` block :

```css
.hasnotebookbar .ui-content.unotoolbutton.selected:not(.has-label):not(.inline) {
	background-color: #e6e6e6b0;
	border-radius: 4px; /*same as .w2ui-toolbar table.w2ui-button*/
}
```

Reply:
Didn't work also the box-shadow stuff didn't work now.